### PR TITLE
SendMailToManager bug fix

### PR DESCRIPTION
**Pull request by Itay4**
## Description
https://github.com/demisto/content/pull/3788

## Required version of Demisto
Any

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by yaakovi** [comment]
why the diff looks like that? try to rebase or something...

File: Documentation/common_server_docs.py [hunk 1]
```diff
@@ -156,18 +157,74 @@ def createPyDocumentation(path, origin, language):
     return reformatPythonOutput(x, origin, language)
 
 
+def createPsDocumentation(path, origin, language):
```
Comment (yardensade) [inline Documentation/common_server_docs.py]: PS is not sort for powershell, should PWSH

File: Documentation/common_server_docs.py [hunk 2]
```diff
@@ -156,18 +157,74 @@ def createPyDocumentation(path, origin, language):
...
+
+    isErrorPS = False
```
Comment (yardensade) [inline Documentation/common_server_docs.py]: PS is not sort for powershell, should PWSH

File: Documentation/common_server_docs.py [hunk 3]
```diff
@@ -156,18 +157,74 @@ def createPyDocumentation(path, origin, language):
...
+        function_doc_list.append(function_doc)
+
+    return function_doc_list, isErrorPS
+
+
 def main(argv):
     jsDoc, isErrorJS = createJsDocumentation('./Documentation/commonServerJsDoc.json', 'CommonServerJs', 'javascript')
     pyDoc, isErrorPy = createPyDocumentation('./Scripts/CommonServerPython/CommonServerPython.py',
                                              'CommonServerPython', 'python')
+    psDoc, isErrorPS = createPsDocumentation('./Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1',
```
Comment (yardensade) [inline Documentation/common_server_docs.py]: PS is not sort for powershell, should PWSH

File: Tests/scripts/constants.py [hunk 4]
```diff
@@ -77,19 +79,22 @@
 
 KNOWN_FILE_STATUSES = ['a', 'm', 'd', 'r'] + ['r{:03}'.format(i) for i in range(101)]
 
-CODE_FILES_REGEX = [INTEGRATION_JS_REGEX, INTEGRATION_PY_REGEX, SCRIPT_PY_REGEX, SCRIPT_JS_REGEX]
+CODE_FILES_REGEX = [INTEGRATION_JS_REGEX, INTEGRATION_PY_REGEX, SCRIPT_PY_REGEX, SCRIPT_JS_REGEX,
+                    INTEGRATION_PS_REGEX, SCRIPT_PS_REGEX]
 
-SCRIPTS_REGEX_LIST = [SCRIPT_YML_REGEX, SCRIPT_PY_REGEX, SCRIPT_JS_REGEX]
+SCRIPTS_REGEX_LIST = [SCRIPT_YML_REGEX, SCRIPT_PY_REGEX, SCRIPT_JS_REGEX, SCRIPT_PS_REGEX]
```
Comment (yardensade) [inline Tests/scripts/constants.py]: PS is not sort for powershell, should PWSH

File: Tests/scripts/hook_validations/structure.py [hunk 5]
```diff
@@ -43,8 +43,10 @@ class StructureValidator(object):
         REPORT_REGEX,
         SCRIPT_PY_REGEX,
         SCRIPT_JS_REGEX,
+        SCRIPT_PS_REGEX,
```
Comment (yardensade) [inline Tests/scripts/hook_validations/structure.py]: PS is not sort for powershell, should PWSH

File: Tests/scripts/hook_validations/structure.py [hunk 6]
```diff
@@ -43,8 +43,10 @@ class StructureValidator(object):
...
         INTEGRATION_JS_REGEX,
         INTEGRATION_PY_REGEX,
+        INTEGRATION_PS_REGEX,
```
Comment (yardensade) [inline Tests/scripts/hook_validations/structure.py]: PS is not sort for powershell, should PWSH

**Comment by Itay4** [comment]
@kirbles19 please review the changelog

**Comment by kirbles19** [comment]
@Itay4 changelog is done.

**Pull request merged**